collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -38,12 +37,38 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
```

Comment:
It's useful to have a `head` element here with the usual summary - not least because Tim and Joanna use this to create the general labels for objects on Digital Bodleian. Although Peter says "sermons" I wonder if "homiliary" would be more helpful given the patristic content